package.json

```diff
@@ -37,5 +37,17 @@
     "now-deploy": "now build --public",
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ]
```

Comment:
I believe we should check which version the Overlay plugin uses so we can deploy with that target in mind. I'll let you know in my tests tonight which version is it. It'd be better to other developers to not be confused with API changes.

Reply:
yeah true, this specific combination of requirements DOES work with overlayplugin which is why i chose it